Model/Feed/Type/Order/RowData.php

```diff
@@ -0,0 +1,50 @@
+<?php
+declare(strict_types=1);
+/**
+ * Copyright © PureClarity. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+namespace Pureclarity\Core\Model\Feed\Type\Order;
+
+use Magento\Sales\Model\Order;
+use Pureclarity\Core\Api\OrderFeedRowDataManagementInterface;
+
+/**
+ * Class RowData
+ *
+ * Handles individual order data rows in the feed
+ */
+class RowData implements OrderFeedRowDataManagementInterface
+{
+    /**
+     * Builds the order data for the order feed.
+     * @param int $storeId
+     * @param Order $order
+     * @return array
+     */
+    public function getRowData(int $storeId, $order): array
+    {
+        $orderData = [];
+
+        $id = $order->getIncrementId();
+        $customerId = $order->getCustomerId();
+        $email = $order->getCustomerEmail();
+        $date = $order->getCreatedAt();
+
+        foreach ($order->getAllVisibleItems() as $item) {
+            $orderData[] = [
+                'OrderID' => $id,
+                'UserId' => $customerId ?: '',
```

Comment:
Why is user id the only one coalesced to an empty string?

Reply:
it's the only field that's possibly not there (if it's a guest order, it'll return null), all the other fields will be there all of the time